src/ape/api/networks.py

```diff
@@ -78,6 +78,7 @@ def __getitem__(self, network_name: str) -> "NetworkAPI":
         return self._try_get_network(network_name)
 
     def __getattr__(self, network_name: str) -> "NetworkAPI":
+        network_name = network_name.replace("_", "-")
```

Comment:
This makes it so `mainnet-fork` can be the network name rather than `mainnet_fork`.